converters/schema.ts

```diff
@@ -0,0 +1,75 @@
+import { ClarityDataSchema, ObjectType } from "../clarity";
+import { hashCode } from "./../src/utils";
+
+export class SchemaManager {
+
+  private schemas = {};
+
+  public getSchema(schemaHash: string): ClarityDataSchema {
+    return this.schemas[schemaHash];
+  }
+
+  public addSchema(schema: ClarityDataSchema): boolean {
+    let schemaHash = hashCode(JSON.stringify(schema));
```

Comment:
>hashCode(JSON.stringify(schema)); [](start = 21, length = 33)

Change to call getSchemaHashcode(). Does same thing as this line.

Reply:
Done, thanks